src/Migrator.EF6.Tools/Migrator.EF6.Tools.csproj

```diff
@@ -5,7 +5,7 @@
 	<PropertyGroup>
 		<Description>A .NetCore CLI tool to enable EF6 migrations in an Asp.Net Core app.</Description>
 		<Authors>Mohammad Rahhal</Authors>
-		<TargetFrameworks>netcoreapp1.0;net46</TargetFrameworks>
+		<TargetFrameworks>netcoreapp1.0;net46;net47</TargetFrameworks>
```

Comment:
You can delete it. I first thought this could be the problem why my project doesn't work. But it's not. Sorry for that.